tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts

```diff
@@ -17,6 +17,18 @@ export default {
             signature: "LitentryMultiSignature",
         },
 
+        TCAuthentication: {
+            _enum: {
+                Web3: "LitentryMultiSignature",
+                Email: "Text",
+            },
+        },
+        TrustedCallAuthenticated: {
```

Comment:
a minor issue: Above there is `TCAuthentication`; here is `TrustedCallAuthenticated`. I know it's coming from RUST code. Would it be better to consistent name pattern? or this is on purpose? @silva-fj

Reply:
Not particular reason, feel free to propose better names. There are a few things we should clean up after the demo